spec/mailers/notify_mailer_spec.rb

```diff
@@ -429,12 +429,6 @@ def create_badge_achievement(user, badge, rewarder)
     it "includes the tracking pixel" do
       expect(email.html_part.body).to include("open.gif")
     end
-
-    it "includes UTM params" do
-      expect(email.html_part.body).to include(CGI.escape("utm_medium=email"))
-      expect(email.html_part.body).to include(CGI.escape("utm_source=notify_mailer"))
-      expect(email.html_part.body).to include(CGI.escape("utm_campaign=export_email"))
-    end
```

Comment:
I removed these tests because we can't track without `@user` set. I think we don't need to track this email but open to changing it!